qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -1005,7 +716,8 @@ def get_num_channels(byte_rep: int) -> int:
         """
         one_channels = tuple(2**i for i in range(16))
         two_channels = (3, 5, 6, 9, 10, 12)
-        four_channels = (255,)
+        four_channels = (15,)
+        eight_channels = (255,)
         sixteen_channels = (65535,)
```

Comment:
Could these lines also be accommodated in the constants.py? (maybe including the elif chain...)

Reply:
yes, i think it's a good idea. i'll see how to refactor this.

Reply:
Done.